src/utils/firstElementfromInstances.ts

```diff
@@ -0,0 +1,10 @@
+export type Mutable<Type> = {
```

Comment:
You can use built-in utility type, Readonly, https://www.typescriptlang.org/docs/handbook/utility-types.html#readonlytype